frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -129,20 +131,20 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
       guestAgent={isGuestAgentConnected(vmi)}
     />
   ) : null;
-
   return (
     <Stack hasGutter>
       {showOpenInNewWindow && consoleType && (
         <StackItem>
           <Button
             variant="secondary"
-            onClick={() =>
+            onClick={() => {
+              localStorage.setItem(`isFullScreenVNC-${vmName}`, 'true');
```

Comment:
this should be done in `connected-vm-console` to make sure everytime we open it (even from other places in the future), the localstorage value will be set properly